Trailheads/src/classes/AccountManagerTest.cls

```diff
@@ -0,0 +1,21 @@
+@isTest
+private class AccountManagerTest {
```

Comment:
Nawiązując do tego co pytałem wyżej, o /Accounts/dupa/contacts.
Zrób test na to - z wywołaniem tego błędnie i potwierdzeniem, że zachowuje się jak należy (np zwróci nulla) a nie, że jebnie błędem. Najpiero nie poprawiaj kodu tylko napisz test (TDD), później zobacz co się dzieje i dopiero ogarnij poprawkę.